src/NUnitConsole/nunit3-console/CommandLineOptions.cs

```diff
@@ -129,7 +127,7 @@ public CommandLineOptions(params string[] args)
         private string workDirectory = null;
         public string WorkDirectory 
         {
-            get { return workDirectory ?? DEFAULT_WORK_DIRECTORY; }
+            get { return workDirectory ?? Environment.CurrentDirectory; }
```

Comment:
The idea of using the static string was to get the field initialized before any other code had a chance to run. It probably doesn't matter so long as we don't do anything else with the CurrentDirectory in the options class.